src/selectBetweenOffsets.ts

```diff
@@ -0,0 +1,83 @@
+import * as vscode from "vscode";
+import { ExtensionHostMessageHandler, MessageType } from "../shared/protocol";
+
+const addressRe = /^0x[a-f0-9]+$/i;
+const decimalRe = /^[0-9]+$/i;
+
+export const showSelectBetweenOffsets = (messaging: ExtensionHostMessageHandler): void => {
+    const startingInput = vscode.window.createInputBox();
```

Comment:
It'd be quite nice for the default value in the starting input to be the currently selected byte. The HexDocument has a `selectionState` that you can read from.

Reply:
I was looking for that but failed. Thanks, I will change it

Reply:
I couldn't access the active HexDocument. So I added a static variable holding that reference. Is this okay or is there a better way to access it? @connor4312

Reply:
You can get it via the `activeDocument` on the `HexEditorRegistry` (created inside activate())

Reply:
Ohh, how did I not see that?? Fixing it now